docs/source/developer_guides/troubleshooting.md

```diff
@@ -135,3 +135,58 @@ model.save_adapter("my_adapter", save_embedding_layers=True)
 For inference, load the base model first and resize it the same way you did before you trained the model. After you've resized the base model, you can load the PEFT checkpoint.
 
 For a complete example, please check out [this notebook](https://github.com/huggingface/peft/blob/main/examples/causal_language_modeling/peft_lora_clm_with_additional_tokens.ipynb).
+
+### Investigating the status of the PEFT model
```

Comment:
I think this is clearer about what exactly you're investigating (layer/model status)

```suggestion
### Check layer and model status
```